src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -226,6 +225,19 @@ private static bool ValidateInvocationPropertiesToLog(TContext context, IEnumera
             return succeeded;
         }
 
+        private bool ValidateOutputFileCanBeCreated(TContext context, string outputFilePath, bool force)
```

Comment:
>force [](start = 98, length = 5)

Kind of an unexpected parameter name. Perhaps forceRequested?

Reply:
I feel you, but the command line option is `--force`, and the property name in the options class(es) is `Force`, so I'd rather leave this one.

---
In reply to: [312250368](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312250368) [](ancestors = 312250368)